commands/moderation/clearmessages.js

```diff
@@ -84,11 +89,11 @@ function clearEmbed(msg, numberDeleted) {
     .setTimestamp()
     .setFooter(`${msg.guild.name}`);
 
-  channel.send(banEmbed);
+  channel.send({embeds: [clearMsgEmbed]});
 }
 
 function clearMessage(msg, numberDeleted) {
   msg.channel.bulkDelete(parseInt(numberDeleted) + 1);
 
-  msg.channel.send(`${numberDeleted} messages were deleted.`);
+  msg.channel.send({content: `${numberDeleted} messages were deleted.`});
```

Comment:
Not a big deal, but worth noting that if all we're sending in the message is a string, then it can be sent as-is, it doesn't need to be inside the options object.

Reply:
Yes, I noticed that either a string or object is ok as an argument to .send. The string is shorter and the object is perhaps more consistent if we add some options to a few of these send methods. We could discuss and perhaps decide on one of these two styles for future PRs.